server/psql.js

```diff
@@ -0,0 +1,39 @@
+const psql = require('../db/psql/postgres.js');
+const { Client } = require('pg');
+
+const client = new Client(psql.config);
+client.connect(() => {console.log('connected to pg client')});
+
+const createQuery = `INSERT INTO products(id,productName,price,totalSales,reviews,inStock,creator,quantity,handmade,descrip,ready,productFrom) VALUES (11000000,'new product',10.00,200,15,true,'sam',15,true,'beautiful wood log','30 Nov 2020 12:00:00 GMT', '30 Nov 2021 12:00:00 GMT') RETURNING *;`
```

Comment:
The `id `column is unique, so just make sure you aren't inserting a row with the same id more than once when you stress test